awx/ui/.linguirc

```diff
@@ -8,7 +8,7 @@
 "compilerBabelOptions": {},
 "fallbackLocales": { "default": "en"},
 "format": "po",
-"locales": ["en","es","fr","ko","nl","zh","ja","zu"],
+"locales": ["en","es","fr","ko","nl","zh","ja","zu","it"],
```

Comment:
@rooftopcellist who's domain is this in? Should the translation stuff here ever be edited by community PRs and is there some policy document we can point to for this?